packages/serverless-api/src/api/utils/api-client.ts

```diff
@@ -1,14 +1,29 @@
 import { ClientConfig } from '../../types';
 
+const regionEdgeMap: { [index: string]: string } = {
+  us1: 'ashburn',
+  au1: 'sydney',
+  ie1: 'dublin',
+  'stage-us1': 'ashburn',
+  'stage-au1': 'sydney',
+};
+
 export function getApiUrl(
   config: ClientConfig,
   product = 'serverless',
   apiVersion = 'v1'
 ): string {
   const configEdge = config.edge || process.env.TWILIO_EDGE;
   const configRegion = config.region || process.env.TWILIO_REGION;
+  const region = configRegion ? `${configRegion}.` : '';
+
+  if (!configEdge && configRegion) {
+    const defaultEdge = regionEdgeMap[configRegion]
```

Comment:
It looks like this will silently ignore invalid regions and replace them with default.  Should we throw an error back to the user here and let them know the region they specified is not a valid region for Serverless API?

Reply:
I remember Dom mentioning, we should let toolkit fallback to the API and be dumb about this.

Reply:
Hm.  Yes, but in this case, it's not exactly expected or desired behavior imo.  If i recall the conversation correctly, we discussed if the user specifies a region we don't have, and then we try to use that region in the URL, then they'd get a 404 back which isn't great or specific, but at least it would fail.  In this case, you specify an invalid region and end up with something deployed to US1, which is kind of unexpected.  @dkundel thoughts?

Reply:
oh wait, im totally misreading!  my fault.  That looks like what this will do.  Okay all good :)